src/CUITe/CUITe for Visual Studio 2015.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>CUITe.VS2015</id>
+    <version>$version$</version>
+    <authors>Suresh Balasubramanian, Matt Gray, Rami Abughazaleh, Mattias Kindborg, and contributors</authors>
+    <title>CUITe for Visual Studio 2015</title>
+    <language>en-US</language>
+    <licenseUrl>https://raw.githubusercontent.com/icnocop/cuite/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/icnocop/cuite</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/icnocop/cuite/master/images/NuGet_32x32.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>CUITe (Coded UI Test enhanced) Framework is a thin layer developed on top of Microsoft Visual Studio Team Test's Coded UI Test engine which helps reduce code, increases readability and maintainability, while also providing a bunch of cool features for the automation engineer.</description>
+    <tags>cuit cuite coded ui test enhanced visual studio 2015 vs2015</tags>
+  </metadata>
+  <files>
+    <file src="bin\Release\VS2015\net40\CUITe.dll" target="lib\net40\CUITe.dll" />
+    <file src="bin\Release\VS2015\net40\CUITe.xml" target="lib\net40\CUITe.xml" />
+    <file src="bin\Release\VS2015\net45\CUITe.dll" target="lib\net45\CUITe.dll" />
+    <file src="bin\Release\VS2015\net45\CUITe.xml" target="lib\net45\CUITe.xml" />
+    <file src="..\CUITe_ObjectRecorder\bin\Release\CUITe_ObjectRecorder.exe" target="tools\CUITe_ObjectRecorder.exe" />
+    <!-- NuGet install files -->
```

Comment:
Yes, we're still using those app.config transformation files to update the binding redirects of test assemblies to a more appropriate version.

I haven't completely finished the integration tests for the Silverlight NuGet packages, so I'll have to do that next. ;)